core/components/molecules/sidesheet/__stories__/CustomHeader.story.jsx

```diff
@@ -66,7 +66,6 @@ export const customHeader = () => {
           </div>
```

Comment:
Please use `<Divider />` instead of using `<Divider></Divider>` as it does not have any children and can be used as a self-closing component.

Reply:
Done